docker/Dockerfile-multi-arch

```diff
@@ -0,0 +1,23 @@
+FROM golang:1.18.3 as build
+WORKDIR /workspace
+ENV GO111MODULE=on \
+    GOPROXY=https://goproxy.cn,direct \
+    BINARY=postgres-operator
+COPY . .
+RUN CGO_ENABLED=0 go build -o build/linux/${BINARY} ./cmd/main.go
+
+ARG base-image
```

Comment:
ARG base-image will need to be defined at the top of the file to be defined as global and will be available in FROM.
I recommend setting the default value for the `base-image=alpine:3.15` in the Dockerfile, which allows it to be used locally with ease.